WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -119,25 +119,29 @@ def update_all
           RegistrationsMailer.notify_registrant_of_accepted_registration(registration).deliver_now
         end
       end
-      flash[:success] = "#{"Registration".pluralize(registrations.length)} accepted! Email #{"notification".pluralize(registrations.length)} sent."
+      flash.now[:success] = "#{"Registration".pluralize(registrations.length)} accepted! Email #{"notification".pluralize(registrations.length)} sent."
```

Comment:
I wouldn't call it unnecessary. JS isn't necessary to edit a simple form, so why make it so?